README.md

```diff
@@ -1 +1,76 @@
-# Portfolio
\ No newline at end of file
+
+
+# Project Name
+
+> Description the project.
+> This is a Portfolio
+
+
+## Built With
+
+- Html
+- css
+- Terminal
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle]()
+- Twitter: [@twitterhandle](https://twitter.com/SchoolsAfrikana)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/okonkwo-echezona-4b92181b0/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/schoolsofafrika)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
[OPTIONAL] remove this second author as you are the only person contributing to this project